source/fundamentals/encrypt-fields.txt

```diff
@@ -45,10 +45,10 @@ uniquely.
 
 The MongoDB manual contains detailed information on the following {+qe+} topics:
 
-- Begin a working {+qe+} example with the :v6.0:`queryable-encryption quick start </core/queryable-encryption/quick-start>`.
-- Learn how to use {+qe+} in the :v6.0:`queryable-encryption fundamentals </core/queryable-encryption/fundamentals>`.
-- Learn how to integrate your {+qe+} implementation with a KMS in the :v6.0:`queryable-encryption tutorials </core/queryable-encryption/tutorials>`.
-- Learn {+qe+} concepts in the :v6.0:`queryable-encryption reference </core/queryable-encryption/reference>`.
+- To get started, see the {+qe+} :v6.0:`Quick Start </core/queryable-encryption/quick-start>`.
```

Comment:
Question: are there any specific reasons to change "Begin a..." to "To get started, see..." here? I don't think any changes are needed either way, just curious.